### PR TITLE
a lot of changes

### PR DESCRIPTION
**Pull request by pyrosliced**
moe do the rest im bored

if it broke dont dm me

### CODE REVIEW

File: options/Addons/Helpful/Overclocked.md
Comment (0xDrMoe) [inline options/Addons/Helpful/Overclocked.md:1]: Missing description

File: options/Impostors/Concealing/Vampire.md
Comment (0xDrMoe) [inline options/Impostors/Concealing/Vampire.md:1]: Missing a few lines of information, my friend

**Review by 0xDrMoe — CHANGES_REQUESTED** [review]

**Comment by 0xDrMoe** [comment]
why was this closed

**Review by 0xDrMoe — APPROVED** [review]

**Comment by Ultradragon005** [comment]
this shit is crazyyyy

**Pull request merged**